src/native/corehost/hostmisc/utils.cpp

```diff
@@ -100,18 +100,22 @@ pal::string_t strip_file_ext(const pal::string_t& path)
 
 bool get_line_from_file(FILE* pFile, pal::string_t& line)
 {
-    std::vector<pal::char_t> buffer;
-    char last_char;
-    while ((last_char = fgetc(pFile)))
+    line = pal::string_t();
+    char buffer[256];
+    while (fgets(buffer, sizeof(buffer), pFile))
```

Comment:
we should add a test with a really long path in the config file - it doesn't need to exist, the failure with the correct string is enough to validate the parsing logic here.